### PR TITLE
Fix simple PE-d issues

### LINKED ISSUE #271: [PE-D][Tester C] Edit does not work as described in UG
edit does not work as described in UG, should be “edit contact...” instead.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: jinhanjinhan/ped#2

### LINKED ISSUE #273: [PE-D][Tester C] Find does not work as described in UG
Find does not work as described in UG, should be "find contact..." i assume? 

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: jinhanjinhan/ped#3

### LINKED ISSUE #278: [PE-D][Tester B] Inconsistent parameter name for add job
UG states that:

But the application is:

Specifically SALARY vs MONTHLY_SALARY and REQUIREMENT vs REQUIREMENTS. Should be this consistent throughout UG and within application.

Command to run: add job.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: gohsl99/ped#2

### LINKED ISSUE #280: [PE-D][Tester A] Edit command
The example that I copied from the UG doesn't work

-------------
Labels: `type.DocumentationBug` `severity.High`
original: WuJinhan1/ped#6

### LINKED ISSUE #286: [PE-D][Tester A] Edit contact UG bug
After modifying the edit command to edit contact 1 I tried one of the fields put as a format under the edit command, the a/ field but it didn't work.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: WuJinhan1/ped#7

### LINKED ISSUE #288: [PE-D][Tester C] Use case not labelled properly
Use case not labelled properly, a cosmetic issue but it also hinders with the reading from a user's perspective. Perhaps can add more spacing and remove the bold on MSS and extensions. Or perhaps can use a numbering system like Use Case 1, Use Case 2, etc…

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jinhanjinhan/ped#8

### LINKED ISSUE #290: [PE-D][Tester C] Planned enhancements are not very well documented, it should be 10 planned enhacements
Planned enhancements are not very well documented, it should be 10 planned enhacements, but from the list, i can't tell how many planned enhancement is that.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: jinhanjinhan/ped#9

### LINKED ISSUE #293: [PE-D][Tester B] Wrong command format for edit given by UG
The command given in the UG is not usable in the application.
UG shows this command format for edit:

But the actual command format for edit (in application) is:

User cannot run the example command from UG in the application and this is a flaw.
Command to reproduce: edit 1 p/91234567 e/johndoe@example.com

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: gohsl99/ped#4

### LINKED ISSUE #297: [PE-D][Tester B] Unmatching parameter in UG and application for edit
UG states that edit requires address:

However, edit in application does not require address, but require role instead:

Run "edit contact" to reproduce.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: gohsl99/ped#5

### LINKED ISSUE #298: [PE-D][Tester C] Format not standardised (PHONE vs PHONE_NUMBER)
Format not standardized (somne parts of UG use PHONE, some parts use PHONE_NUMBER), so not well formatted. Might get easily confused with whether PHONE refers to a phone model or something else. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jinhanjinhan/ped#13

### LINKED ISSUE #299: [PE-D][Tester B] Command format for find stated in UG is not applicable in the application
Command format for UG states that these parameters are required for find:

But the application requires a different command format:

Command to reproduce: find

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: gohsl99/ped#6

### LINKED ISSUE #316: [PE-D][Tester E] Find method should accept numbers because name accepts numbers
What if I named the guy John 1 John 2 and needed to find John 2?

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: starchypotatocode/ped#15

### LINKED ISSUE #341: [PE-D][Tester C] Screen job command does not screen users that already have a job, but not documented in UG.
Screen job command does not screen users that already have a job, but not documented in UG. As a user, i initally thought it should screen all users who have the role, based on solely reading the UG, but it doesn't seem like it.

contact 1 is matched as seen below

but when i tried to `screen job 1` as seen below, it doesnt show contact 1, although i would assume it should show

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: jinhanjinhan/ped#24

### LINKED ISSUE #346: [PE-D][Tester E] Missing command summary
list all is not present but should be. however this is low severity because the error message does show list all.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: starchypotatocode/ped#25

### LINKED ISSUE #350: [PE-D][Tester E] Incorrect UG documentation
Low as it's super obvious if you try and follow the UG

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: starchypotatocode/ped#28

### LINKED ISSUE #356: [PE-D][Tester D] Filtering based on job listing
In your user stories, one high priority feature was to be able to filter contacts based on job listing, however I am not sure how your app allows for that. Currently, I am only able to filter based on the contact's name.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: karthickkc/ped#10

### PR DESCRIPTION
**Pull request by Quasant**
You can choose to unmark any of the issues, if you think they are not satisfactorily done.
 
### Edit issues (all within here are similar, but do still look through)
- [x] Closes #286 
- [x] Closes #293
- [x] Closes #297 
- [x] Closes #271 
- [x] Closes #280 
- [x] Closes #350 
- [x] Closes #316 
  - Moved from rejected to closed
- Related but still unsolved issues are #314 and #320

### Find, list, screen, misc issue
- [x] Closes #273 
- [x] Closes #299
- [x] Closes #346 - for list all
- [x] Closes #341 
- [x] Closes #356 
- [x] Closes #290, already fixed
- [x] Closes #298 
- [x] Closes #278 
- [ ] #281
  - I changed this back to Skill, and considering if we should change Skills to skill in app instead.
- [x] Closes #288 and #335

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -245,11 +245,6 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
-### \[Proposed\] Data archiving
-
-_{Explain here how the data archiving feature will be implemented}_
-
-
```
Comment (Quasant) [inline docs/DeveloperGuide.md:252]: Removing it for now, unless someone has objection.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -494,20 +489,8 @@ testers are expected to do more *exploratory* testing.
 
 ## **Appendix: Planned Enhancements**
 
+Team size: 5
```
Comment (Quasant) [inline docs/DeveloperGuide.md:492]: Requirement gotten from: https://nus-cs2103-ay2425s1.github.io/website/admin/tp-w11.html

We have 10 planned enhancements as a shield.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -494,20 +489,8 @@ testers are expected to do more *exploratory* testing.
...
-  * Support job quota
-  * Smart select company if not found when creating job listing (find & return 1st match)
   * Support screening candidates via more advanced requirement (salary range, number of skills)
-  * Sorting capability of job list
-* Company
-  * Edit a company
-  * Locating company by name
-  * Enhance company contact details by having contacts as company liaises 
-* Archiving capability
-* Set a contact as potential candidate of a job
```
Comment (Quasant) [inline docs/DeveloperGuide.md:513]: For these deleted enhancements, which ones do you think will be possible feature flaws highlighted by users?

Comment (Green-Tea-123) [reply docs/DeveloperGuide.md:513]: definitely edit and find for the other 2 entities

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Quasant) [reply docs/DeveloperGuide.md:252]: Removed

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Quasant) [reply docs/DeveloperGuide.md:513]: Yup seems like they were highlighted. On monday, recommend the meeting to settle this.

**Pull request merged**